components/elevation/src/vwc-elevation.scss

```diff
@@ -28,5 +28,8 @@ $vvd-elevation-border-radius: --vvd-elevation-border-radius;
 .vwc-elevation {
 	width: fit-content;
 	border-radius: var(#{$vvd-elevation-border-radius}, 6px);
-	contain: content;
+}
+
+::slotted(*) {
+	border-radius: var(#{$vvd-elevation-border-radius}, 6px);
 }
```

Comment:
what if it's just an icon or text that will be assigned to slot?
this is unexpected as we have no control over the content.

need to brain storm again